src/odemis/driver/tmcm.py

```diff
@@ -418,6 +421,44 @@ def __init__(self, name, role, port, axes, ustepsize, address=None,
             self._updateTemperatureVA() # make sure the temperature is correct
             self._temp_timer.start()
 
+    def _update_ref(self):
+        """
+        It updates the global parameter AREF_USER_VAR with the axes that are referenced
+        """
+        # AREF_USER_VAR is a general purpose 32-bit variable in bank 2. It is used to store the axes that are already referenced.
+        # Up to 256 variables are available, only the first 56 can be stored permanently in EEPROM. #117 is randomly chosen between 56-255.
+        # In case the TMCM controller is turned off, the global parameter is automatically reset.
```

Comment:
"In case the TMCM controller is turned off, or the main motor power is turned off/on, the variable is automatically reset to 0. We use this behaviour to detect whether the axes could have been moved without the device control."